src/screens/op-rewards/mocks/getMockOpData.ts

```diff
@@ -0,0 +1,114 @@
+// Mock data is based on schema provided by Alexey
+const MOCK_DATA = {
+  meta: {
+    title: 'Optimism Rewards',
+    // Unix timestamp in UTC. "Next Airdrop" in Figma.
+    next_distribution: 1673341200,
+    distribution: {
+      next: 1673341200,
+      rewards_total: 28000.0,
+      rewards_left: 25597.248,
+    },
+    // Possible statuses are: `ongoing`, `paused`, `finished`
+    status: 'ongoing',
+    // Unix timestamp in UTC. "Days left" in Figma.
+    end: 1675983600,
+    token: {
+      asset: {
+        asset_code: '0x4200000000000000000000000000000000000042',
+        decimals: 18,
+        icon_url:
+          'https://rainbowme-res.cloudinary.com/image/upload/v1668486694/assets/optimism/0x4200000000000000000000000000000000000042.png',
+        name: 'Optimism',
+        network: 'optimism',
+        symbol: 'OP',
+      },
+    },
+  },
+  earnings: {
+    total: {
+      usd: 229.25,
+      token: 201.502,
+    },
+    multiplier: {
+      amount: 2,
+      breakdown: [
+        {
+          amount: 1.0,
+          qualifier: 'Base',
+        },
+        {
+          amount: 0.25,
+          qualifier: 'Has previously swapped in Rainbow',
```

Comment:
not a big an issue for this case but wonder how we should be doing i18n for strings like this